### ISSUE TITLE
GUI Issue in Flex Display

### ISSUE THREAD

**Issue by jonathandmoyer** [PardeeCenterDU/IFs-Issues-Tracking#62]
Please move Explain Initials to a sub-option within List Features.

There is also an issue of the language used in "List Features".  It's not clear what a "List" is.  The category that contains the "Lists" is called "Displays".  The "Displays" are actually variables and parameters and we should probably eventually adjust our language there as well, but at the very least we should change "List Features" to "Display Features" and then change the sub-categories from "List" to "Display".  I think the complication of changing this to "Display" is because in Self Managed Display we call "Display Lists" list for Geographically Fixed Displays (which is not used) and "Variable Lists" those created for Flex Display.  We should also change the name for "Display List" to "Geographically Fixed Variables and Parameters" and "Variable Lists" to "Flex Display Variables and Parameters".  

Lots here and probably worth a conversation, but also important for us to keep refining this language.

**Comment by PardeeCenterIFs** [comment]
I have moved menu Explain Initials under List Features.

Let's discuss the Language issue in the Pod. I don't like the word Display, and we probably changed it some time ago, I think it used to be List, to be consistent with the language everywhere else. Anyway it's a list of variables and parameters, that's what it is, we could call it a set, or something else, but Display sounds like a graph.

**Comment by jonathandmoyer** [comment]
I agree they are lists or sets, but the page is full of lists and sets.  There needs to be something more specific I think.  Happy to discuss it in the pod.
